CalPatRec/src/CalHelixFinderAlg.cc

```diff
@@ -2092,8 +2094,19 @@ namespace mu2e {
       printInfo(Helix);
     }
 
+    circleHits = Helix._sxy.qn();
+
     rc = doLinearFitPhiZ(Helix, HitInfo_t(StrawId::_ntotalfaces-1,0,-1), useIntelligentWeight);
 
+    phiHits = Helix._szphi.qn();
+
+    nHitsRatio = circleHits/(phiHits + 1e-6);
+
+    if (_diag > 0) {
+      Helix._diag.nHitsRatio = nHitsRatio;
+    }
+
+    if (nHitsRatio > _maxNHitsRatio) goto PATTERN_RECOGNITION_END;
     //2017-10-05 Gianipez added the following line to make some tests
```

Comment:
Is this comment still relevant? Git will keep track of changes and who made them, but it sounds like a test may have been left in?

Reply:
What was tested became permanent. Please don't remove the compiled code.

Reply:
I agree with Dave that a comment like this is not helpful.  If it's important to explain why the test is there then explain it.  If not, the test stands on its own.

Reply:
I agree with removing the comment. I said please don't remove the compiled code, aka, line 2111.